WordPress/Classes/Services/AccountService.h

```diff
@@ -134,7 +144,9 @@ extern NSString *const WPAccountEmailAndDefaultBlogUpdatedNotification;
 
  @param account WPAccount to be updated
  */
-- (void)updateUserDetailsForAccount:(WPAccount *)account success:(void (^)(void))success failure:(void (^)(NSError *error))failure;
+- (void)updateUserDetailsForAccount:(WPAccount *)account
+                            success:(nullable void (^)(void))success
+                            failure:(nullable void (^)(NSError *error))failure;
```

Comment:
I was always omitted them since Xcode autocomplete doesn't carry them over — and it does show up correctly annotated in autocomplete: 
<img width="536" alt="screen shot 2017-10-07 at 00 25 24" src="https://user-images.githubusercontent.com/1594081/31300799-08f804d4-aaf6-11e7-879f-50698e7ef67e.png">

I guess it's easier to miss it then if it's just few lines above, added :)